src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,22 @@
+package duke.commands;
+
+
+/**
+ * Command enum.
```

Comment:
Consider describing what kind of enumerations there are here (ie LIST, DONE, DELETE, TERMINATE, TASK, FIND, INVALID) and what they are used for